### PR TITLE
Remove jQuery

### PR DESCRIPTION
**Pull request by debbieellis7**
Issue: #54 
Remove jQuery as dependency

## Checklist:
- [x] I've tested the code.
- [x] My code is easy to read, follow, and understand
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation / docblocks.

### CODE REVIEW

File: assets/js/Previewer/Generator.js [hunk 1]
```diff
@@ -100,13 +101,12 @@ export default class {
...
    */
   maybeReloadPreview () {
-
     /*
      * If the viewer hasn't already been initialised, OR
      * the form has been updated AND the previewer container is in the browser viewpoint
      * then we'll generate a new preview
      */
-    if (!this.viewer.doesViewerExist() || (window['gf_submitting_' + this.$form.data('fid')] != true && this.formUpdated && this.isContainerInViewpoint())) {
+    if (!this.viewer.doesViewerExist() || this.formUpdated && this.isContainerInViewpoint()) {
```
Comment (jakejackson1) [inline assets/js/Previewer/Generator.js:109]: We need to keep the `window['gf_submitting_' + this.$form.data('fid')]` check here, as it determines if Gravity Forms is currently in the middle of submitting the form.

File: assets/js/Previewer/Generator.js [hunk 2]
```diff
@@ -128,22 +128,23 @@ export default class {
    * @since 0.1
    */
   async generatePreview () {
-
     /*
      * Only reload the PDF if our container isn't currently hidden AND this function isn't already in progress
      */
-    if (this.$container.is(':visible') && !this.updateInProgress) {
+    if (this.container !== null && !this.updateInProgress) {
```
Comment (jakejackson1) [inline assets/js/Previewer/Generator.js:134]: Checking if `container` is not null is not the same as checking if the `container` is currently in the viewpoint. Will need to mimic the `is(:visible)` call.

File: assets/js/main.js [hunk 3]
```diff
@@ -35,35 +36,25 @@ require('../scss/previewer.scss')
  * @since 0.1
  */
 $(document).bind('gform_post_render', function (e, formId) {
-  /* Don't run if currently submitting the form */
-  if (window['gf_submitting_' + formId]) {
```
Comment (jakejackson1) [inline assets/js/main.js:39]: We need to retain this check so the function doesn't run if GF is in the middle of submission

File: assets/js/main.js [hunk 4]
```diff
@@ -35,35 +36,25 @@ require('../scss/previewer.scss')
...
-    return
-  }
-
-  let $form = $('#gform_' + formId)
-
-  /* Try match a slightly different mark-up */
-  if ($form.length == 0) {
-    $form = $('#gform_wrapper_' + formId).closest('form')
```
Comment (jakejackson1) [inline assets/js/main.js:47]: We need to retain this fallback check for better third-party GF plugin support.

**Comment by debbieellis7** [comment]
Moved to #68

**Pull request closed without merging**